docs/tsc/meetings/2019-03-11.md

```diff
@@ -0,0 +1,85 @@
+Mar 11, 2019
+
+Host: Michael Dolan
+
+Rotating Secretary: Carl Rand
+
+Attendees:
+  * [X] Mark Boorer (_TSC_) - Industrial Light & Magic
+  * [ ] Sean Cooper (_TSC_) - DNEG
```

Comment:
```suggestion
  * [X] Sean Cooper (_TSC_) - DNEG
```

Sean was present.

Reply:
Correction made in commit 519694f4b44c3d9a5e57f5084f221329b4734236